testing/tests/DevExpress.exporter/exceljsParts/exceljs.pivotGrid.tests.js

```diff
@@ -1616,6 +1616,92 @@ QUnit.module('Scenarios', moduleConfig, () => {
+        { format: 'second', expectedFormat: '[$-9]ss', expectedText: '09' },
+        { format: 'minute', expectedFormat: '[$-9]mm', expectedText: '09' },
+        { format: 'hour', expectedFormat: '[$-9]HH', expectedText: '09' },
+        { format: 'day', expectedFormat: '[$-9]d', expectedText: '9' },
+        { format: 'month', expectedFormat: '[$-9]MMMM', expectedText: 'October' },
+        { format: 'year', expectedFormat: '[$-9]yyyy', expectedText: '2019' },
+        { format: 'quarter', expectedFormat: '[$-9]\\QM', expectedText: 'Q4' },
+        { format: 'monthAndDay', expectedFormat: '[$-9]MMMM d', expectedText: 'October 9' },
+        { format: 'monthAndYear', expectedFormat: '[$-9]MMMM yyyy', expectedText: 'October 2019' },
+        { format: 'quarterAndYear', expectedFormat: '[$-9]\\QM yyyy', expectedText: 'Q4 2019' },
+        { format: 'shortDate', expectedFormat: '[$-9]M\\/d\\/yyyy', expectedText: '10/9/2019' },
+        { format: 'shortTime', expectedFormat: '[$-9]H:mm AM/PM', expectedText: '9:09 AM' },
+        { format: 'longDateLongTime', expectedFormat: '[$-9]dddd, MMMM d, yyyy, H:mm:ss AM/PM', expectedText: 'Wednesday, October 9, 2019, 9:09:09 AM' },
+        { format: 'shotDateShortTime', expectedFormat: '[$-9]ssAM/PMSS\\o\\r\\t\\T\\im\\e', expectedText: '99otDAMte09ortTi9e' },
+        { format: 'longDate', expectedFormat: '[$-9]dddd, MMMM d, yyyy', expectedText: 'Wednesday, October 9, 2019' },
+        { format: 'longTime', expectedFormat: '[$-9]H:mm:ss AM/PM', expectedText: '9:09:09 AM' },
+        { format: 'dayOfWeek', expectedFormat: '[$-9]dddd', expectedText: 'Wednesday' },
+        { format: 'yyyy-MM-dd', expectedFormat: '[$-9]yyyy-MM-dd', expectedText: '2019-10-09' }
+    ].forEach((format)=> {
+        const dateValue = new Date(2019, 9, 9, 9, 9, 9, 9);
+
+        [
+            { value: dateValue, expectedPivotCellValue: dateValue, expectedExcelCellValue: new Date(Date.UTC(2019, 9, 9, 9, 9, 9, 9)) },
+            { value: '2019-10-09T00:00:00', expectedPivotCellValue: new Date(2019, 9, 9), expectedExcelCellValue: new Date(Date.UTC(2019, 9, 9)) },
+            { value: '2019/10/9', expectedPivotCellValue: new Date(2019, 9, 9), expectedExcelCellValue: new Date(Date.UTC(2019, 9, 9)) },
+            { value: dateValue.getTime(), expectedPivotCellValue: dateValue, expectedExcelCellValue: new Date(Date.UTC(2019, 9, 9, 9, 9, 9, 9)) }
+        ].forEach((date) => {
+            QUnit.test(`Export [string x string x number] with date format '${format.format}', cell.value: ${JSON.stringify(date.value)}`, function(assert) {
+                const done = assert.async();
+
+                const ds = {
+                    fields: [
+                        { area: 'row', dataField: 'row1', dataType: 'string' },
+                        { area: 'column', dataField: 'col1', dataType: 'string' },
+                        { area: 'data', summaryType: 'max', dataType: 'date', dataField: 'date', format: format.format }
```

Comment:
What about 'date' type in column/row areas?

                        { area: 'row', dataField: 'row1', dataType: 'date' },
                        { area: 'column', dataField: 'col1', dataType: 'date' },

Can it be shown as a 'simple date' without Y/Q/M ?

Reply:
I checked it. 'Row' and 'Column' areas support the predefined custom format too. There are additional tests scenarios into the merge. Please, look at again